dask_cuda/dask_cuda_worker.py

```diff
@@ -192,9 +192,17 @@
     type=str,
     default=None,
     help="When None (default), 'UCX_NET_DEVICES' will be left to its default. "
-    "Otherwise, it must be a non-empty string with the interface name. Normally "
-    "used only with --enable-infiniband to specify the interface to be used by "
-    "the worker, such as 'mlx5_0:1' or 'ib0'.",
+    "Otherwise, it must be a non-empty string with the interface name, such as "
```

Comment:
This a rather lengthy message for CLI output.  What do you think about redirecting them to documentation on how to properly use ?

Reply:
Do you mean linking to https://github.com/rapidsai/dask-cuda/blob/branch-0.14/dask_cuda/local_cuda_cluster.py#L98-L101, or do you have something else in mind that's immediately available?

Reply:
Nothing that's immediately available.  But we could add this to a configuration section on how to properly use IB within dask-cuda.  If you would prefer to keep this as I can submit a follow up PR and we can discuss there

Reply:
Certainly there's a lot to be documented, and this is one of those things. The idea here is just not to leave that uncovered for now. Personally, I would leave it as is just to provide a very descriptive information of what's supposed to be used -- not that I've seen anybody who actually read this anyway -- and address this together with the upcoming documentation improvements that we're working on.

Reply:
Feel free to disagree with me though and suggest a different solution. I'm considering this as temporary for now.

Reply:
Agreed with all that